common/test/acceptance/tests/discussion/test_discussion.py

```diff
@@ -425,6 +432,26 @@ def test_discussion_blackout_period(self):
         self.assertFalse(self.thread_page.is_comment_deletable("comment1"))
         self.assertFalse(self.thread_page.is_comment_deletable("comment2"))
 
+    def test_dual_discussion_module(self):
+        """
+        Scenario: There should be two inline discussion module in one courseware unit
+        Now we have two new post button each of discussion module. When we click on
+        one discussion module new post button it should only expand that discussion
+        module and show new post form of that module. It shouldn't open th new post
+        form of other discussion module.
+        """
+        self.discussion_page.wait_for_page()
+        self.additional_discussion_page.wait_for_page()
+        self.discussion_page.click_new_post_button()
+        with self.discussion_page.handle_alert():
+            self.discussion_page.click_cancel_new_post()
+        self.additional_discussion_page.click_new_post_button()
+        self.assertFalse(self.discussion_page._is_element_visible(".new-post-article"))
+        with self.additional_discussion_page.handle_alert():
+            self.additional_discussion_page.click_cancel_new_post()
+        self.discussion_page.click_new_post_button()
+        self.assertFalse(self.additional_discussion_page._is_element_visible(".new-post-article"))
```

Comment:
self.discussion_page.click_new_post_button() check if the newPostForm is opened for discussion_page that is one discussion module. where as self.additional_discussion_page._is_element_visible is checking that it doesn't effect the second module that is additional_discussion_page.